server/views/review/curfewAddress.pug

```diff
@@ -0,0 +1,31 @@
+extends ../layout
+
+block content
+
+    - var errorsOnPage = Object.keys(errorObject).length !== 0;
+
+    include ../includes/backToTaskList
+
+    if errorsOnPage
+        div.error-summary(role="alert" aria-labelledby="error-summary-heading-example-1" tabindex="-1")
+            h2.heading-medium.error-summary-heading.center.noMargin#error-summary-heading-example-1
+                | There are unanswered questions - see below
+
+    h1.heading-large Licence details
+    div.largeMarginBottom
```

Comment:
Don't know why these pages had to change but if they did, is this the right heading?